### DISCUSSION TITLE
[Feature Request]: Apache

### DISCUSSION THREAD

**Discussion by Tklaversma** [beyondcode/herd-community#490] [Ideas]
Would be nice if we could chose Apache instead of Nginx.

I think most developers are dealing with clients still running Apache and they want to mimic those production servers locally.

**Comment by Tklaversma** [comment]
@sschlein you got any update regarding this feature request?

**Comment by EusebiuOprinoiu** [comment]
I would love to see Apache available in Laravel Herd as well. (preferably with the option to switch between Nginx and Apache at any time)
